### PR TITLE
Docker build attempt 1

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

**Comment by MrCreosote** [comment]
docker-compose up allows saving and getting files, confirming mongo & minio connections.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+RUN chmod a+x /kb/deployment/blobstore/blobstore
+
+# The BUILD_DATE value seem to bust the docker cache when the timestamp changes, move to
+# the end
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-url="https://github.com/kbase/blobstore.git" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.schema-version="1.0.0-rc1" \
+      us.kbase.vcs-branch=$BRANCH \
+      maintainer="Steve Chan sychan@lbl.gov"
```
Comment (sychan) [inline Dockerfile]: Please put your own email address here.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# These ARGs values are passed in via the docker build command
+ARG BUILD_DATE
+ARG VCS_REF
+ARG BRANCH=develop
+
+# Dockerize installation
+RUN apk add wget ca-certificates
+ENV DOCKERIZE_VERSION v0.6.1
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
```
Comment (sychan) [inline Dockerfile]: Don't use the jwilder binary, it doesn't include our site specific features. For example, none of the dockerize changes you or keith have requested are in there. Use the KBase version:
https://github.com/kbase/dockerize

Comment (MrCreosote) [reply Dockerfile]: Huh, I just copied from @jayrbolton 's cache service. Jay FYI

File: Dockerfile (hunk 1 above)
Comment (MrCreosote) [reply Dockerfile]: done

File: Dockerfile (hunk 2 above)
Comment (MrCreosote) [reply Dockerfile]: done

**Comment by MrCreosote** [comment]
@sychan ready for re-review / merge

**Comment by MrCreosote** [comment]
Codecov seems to have gotten confused, since no code files changed and yet it's complaining about coverage

**Comment by sychan** [comment]
Looks good

**Pull request merged**